compatibility/bazel_tools/testing.bzl

```diff
@@ -195,11 +199,30 @@ excluded_test_tool_tests = [
         ],
     },
     {
-        "end": before_grpc_error_code_breaking_change,
+        "end": last_nongranular_test_tool,
```

Comment:
Is `before_grpc_error_code_breaking_change` still used anywhere?

Reply:
Yes it’s used for the other exclusion: current ledger api test tool against older platforms.

Reply:
Yup, a few lines above.